zubhub_frontend/zubhub/src/App.js

```diff
@@ -175,555 +134,401 @@ function App(props) {
+              </PageWrapper>
+            )}
+          />
+
+          <Route
+            path="/teams/:groupname/followers"
+            render={routeProps => (
+              <PageWrapper {...routeProps} {...props}>
+                <LazyImport LazyComponent={TeamFollowers} {...routeProps} {...props} />
+              </PageWrapper>
+            )}
+          />
+
+          <Route
+            path="/teams/:groupname/members"
+            render={routeProps => (
+              <PageWrapper {...routeProps} {...props}>
+                <LazyImport LazyComponent={TeamMembers} {...routeProps} {...props} />
+              </PageWrapper>
+            )}
+          />
+
+          <Route
+            path="/creators/:username/following"
+            render={routeProps => (
+              <PageWrapper {...routeProps} {...props}>
+                <LazyImport LazyComponent={UserFollowing} {...routeProps} {...props} />
+              </PageWrapper>
+            )}
+          />
+
+          <Route
+            path="/teams/all"
+            render={routeProps => (
+              <PageWrapper {...routeProps} {...props}>
+                <LazyImport LazyComponent={AllTeams} {...routeProps} {...props} />
+              </PageWrapper>
+            )}
+          />
+
+          <ProtectedRoute path="/team" component={Teams} {...props} />
+
+          <Route
+            path="/creators/:username/members"
+            render={routeProps => (
+              <PageWrapper {...routeProps} {...props}>
+                <LazyImport LazyComponent={GroupMembers} {...routeProps} {...props} />
+              </PageWrapper>
+            )}
+          />
+
+          <Route
+            path="/creators/:username/add-members"
+            render={routeProps => (
+              <PageWrapper {...routeProps} {...props}>
+                <LazyImport LazyComponent={AddGroupMembers} {...routeProps} {...props} />
+              </PageWrapper>
+            )}
+          />
+
+          <Route
+            path="/creators/:username"
+            render={routeProps => (
+              <PageWrapper {...routeProps} {...props}>
+                <LazyImport LazyComponent={Profile} {...routeProps} {...props} />
+              </PageWrapper>
+            )}
+          />
+
+          <Route
+            path="/account-status"
+            render={routeProps => (
+              <PageWrapper {...routeProps} {...props}>
+                <LazyImport LazyComponent={AccounStatus} {...routeProps} {...props} />
+              </PageWrapper>
+            )}
+          />
+
+          <Route
+            path="/teams/:groupname"
+            render={routeProps => (
+              <PageWrapper {...routeProps} {...props}>
+                <LazyImport LazyComponent={Team} {...routeProps} {...props} />
+              </PageWrapper>
+            )}
+          />
+
+          <ProtectedRoute path="/profile" component={Profile} {...props} />
```

Comment:
using the `ProtectRoute` component. This is where I made all the changes. The rest are prettier indention updates. Am also hoping to configure `prettier` with `eslint` for code uniformity
cc @tuxology